package.json

```diff
@@ -4,14 +4,18 @@
   "private": true,
   "dependencies": {
     "@apollo/react-hooks": "^3.1.3",
-    "@gnosis.pm/safe-apps-sdk": "^0.1.1",
-    "@gnosis.pm/safe-react-components": "^0.1.3",
+    "@gnosis.pm/safe-apps-sdk": "https://github.com/gnosis/safe-apps-sdk.git#25a1afe",
+    "@gnosis.pm/safe-react-components": "https://github.com/gnosis/safe-react-components.git#be51ca5",
```

Comment:
same here ;)

Reply:
We could, but we are not using components in that way, it's pretty unstable so we decided to use commits references instead, to avoid releasing all the time. However, we don't release for a while, so perhaps it's a good opportunity to do it.